Apps/MaybeFinance/config.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "MaybeFinance",
+    "version": "0.1.0",
```

Comment:
Change to: `sha-9522a191de1f258d76ed996ba1c84e3a0629dd80` or BigBearCI will not pass.